packages/webapp/src/Routes.jsx

```diff
@@ -316,6 +316,114 @@ const UnknownRecord = React.lazy(() =>
   import('./containers/ErrorHandler/UnknownRecord/UnknownRecord'),
 );
 
+const FinancesRoutes = () => (
+  <Switch>
+    <Route path="/finances/transactions" exact component={Finances} />
```

Comment:
I see that it is scoped in figma this way -- I am just surprised we are getting rid of the concept of a "Finances home" / "Dashboard" with a route of just /finances

Reply:
True I don't know the right way to do it, since I can potentially see more data visualizations in the future being something Kevin + David might think about. I still personally like keeping the `/finances` home url especially since the title is "finances overview" on the page itself not just transactions. But easily reversed in the future too so I am good with it.